home-manager-module.nix

```diff
@@ -35,6 +35,11 @@ in
         default = "*-*-* 00:00:00";
       };
     };
+    podman = mkOption {
+      type = types.package;
+      default = config.virtualisation.podman.package;
```

Comment:
This assumes HM module will be used with NixOS. Standalone installation will fail.

Reply:
Looks like it fails even in NixOS environment... Maybe HM module can use `pkgs.podman` as the default value 🤔